backend/tracim_backend/config.py

```diff
@@ -414,6 +415,21 @@ def _load_global_config(self) -> None:
             cast_func=AuthType,
             do_strip=True,
         )
+        self.USER__READ_ONLY_FIELDS: typing.Dict[str, typing.List[str]] = {}
+        for auth_type in self.AUTH_TYPES:
+            read_only = string_to_unique_item_list(
```

Comment:
`read_only` is a list of strings, right? So you should better name it `readonly_field_list`

Reply:
Resolved in last commit